### PR TITLE
Development/ideaspace

### PR DESCRIPTION
**Pull request by chadcrotchett**

### CODE REVIEW

**Review by dbradham — APPROVED** [review]
looks good to me!

**Review by chungthuang — APPROVED** [review]

File: apps/app/pages/ideaspace/terms.js
Comment (chungthuang) [inline apps/app/pages/ideaspace/terms.js:1]: The codeowner for anything under `apps/app` is https://github.com/dev-launchers/dev-launchers-platform/blob/master/.github/CODEOWNERS#L17, that's why you didn't have enough approvals to merge.

**Pull request merged**
